### PR TITLE
Allow us to locally run tests on Chrome Canary.

### PR DESCRIPTION
**Pull request by nainar**
This is useful for testing more experimental work in Chrome. For e.g Houdini APIs like AnimationWorklet. 

Next PR will add support for specifying flags to pass to Chrome (if I can pull it off)

### CODE REVIEW

**Comment by nainar** [comment]
Ping @erwinmombay

File: build-system/tasks/runtime-test.js [hunk 1]
```diff
@@ -173,6 +176,7 @@ function printArgvMessages() {
         cyan(argv.grep) + '".',
     coverage: 'Running tests in code coverage mode.',
     headless: 'Running tests in a headless Chrome window.',
+    'chrome_canary': 'Running tests on Chrome Canary.',
```
Comment (rsimha) [inline build-system/tasks/runtime-test.js]: nit: For easy maintenance, move this up to the top, where the other local browser options are listed.

File: build-system/tasks/runtime-test.js [hunk 2]
```diff
@@ -719,6 +723,7 @@ gulp.task('test', 'Runs tests', preTestTasks, function() {
     'a4a': '  Runs all A4A tests',
     'coverage': '  Run tests in code coverage mode',
     'headless': '  Run tests in a headless Chrome window',
+    'chrome_canary': 'Running tests on Chrome Canary.',
```
Comment (rsimha) [inline build-system/tasks/runtime-test.js]: nit: Move this next to the other browsers too.

wording nit: This gets printed for `gulp help`, so reword it to `Runs tests on Chrome Canary` to go with all the other help messages.

File: build-system/tasks/runtime-test.js [hunk 3]
```diff
@@ -60,6 +60,9 @@ function getConfig() {
   if (argv.ie) {
     return Object.assign({}, karmaDefault, {browsers: ['IE']});
   }
+  if (argv.chrome_canary) {
+    return Object.assign({}, karmaDefault, {browsers: ['ChromeCanary']});
```
Comment (rsimha) [inline build-system/tasks/runtime-test.js]: I assume you read https://github.com/karma-runner/karma-chrome-launcher#configuration, which says you can use chromium too. (Not sure if that's useful for AMP development.)

Comment (nainar) [reply build-system/tasks/runtime-test.js]: Yup, I think we want to keep our reliance on Chromium minimum, since there are some (low risk albeit) mods from Chromium -> Chrome.

File: build-system/tasks/runtime-test.js (hunk 1 above)
Comment (nainar) [reply build-system/tasks/runtime-test.js]: Done.

File: build-system/tasks/runtime-test.js (hunk 2 above)
Comment (nainar) [reply build-system/tasks/runtime-test.js]: Moved and phrasing change made.

**Comment by rsimha** [comment]
This PR LGTM.

/cc @danielrozenberg to comment on the visual diffs. There appears to have been a one pixel shift in several pages.

**Comment by danielrozenberg** [comment]
Percy recently changed something on their end and emailed me about it, looks like it might be related. I'll approve the diffs
